packages/ui-components/src/chat/chat.stories.tsx

```diff
@@ -37,23 +37,12 @@ const messages = [
             'Do we want to consider some of the management of metacell / template registries as a UI/UX flow occurring outside of Studio itself? i.e. it is a platform level user journey?',
         timestamp: '12:01',
```

Comment:
It is a timestamp, when you add messages we get a timestamp, here was just for the example when I didn't type it in full as I just wanted to check on the scroll behaviour, but I can update it with what it would normally have.